nansat/mappers/mapper_netcdf_cf.py

```diff
@@ -142,7 +142,7 @@ def _time_count_to_np_datetime64(self, time_count, time_units=None):
             raise Exception('Check time units..')
         return tt
 
-    def _band_list(self, gdal_dataset, gdal_metadata, netcdf_dim={}, bands=[], *args, **kwargs):
```

Comment:
Why is that removed? It seems that the 'revert commit' 53e91c7 does something harmful here and maybe in some other files. Can you double check, @mortenwh ?

Reply:
this is strange... I did an erroneous merge with master, undid it, and merged with develop instead... So it hasn't worked as intended?

Reply:
Both versions a478ab6398da095f90224c84fbe898a631e91ab0 and 53e91c7febf760d8f59f25e8cd6c170b311926c5 lack `*args` and `**kwargs`. None of these are used in the function so maybe not needed?

There is no difference between a478ab6398da095f90224c84fbe898a631e91ab0 and 53e91c7febf760d8f59f25e8cd6c170b311926c5. But, when I look at the blame in github, it seems `*args` and `**kwargs` were added in a commit in master. Since it didn't exist in issue206_clean_up_dicstrings_and_add_doctests it has been removed when the merge with master was reverted. Since the history is kept, this removal happens after the addition in master, and therefore it is not added when we later merge with develop (which has the same changes from master) instead.... Does this make sense? 

In other words: Since the merge still remains in the history, trying to merge that same branch in again later won't include any commits from the merging branch prior to the reverted merge. 

I `git revert 53e91c7`...